data/strings/strings.txt

```diff
@@ -16382,6 +16382,143 @@
     zh-Hans = 运营者
     zh-Hant = 營運者
 
+  [editor_category_unsuitable_title]
+    tags = android
+    en = Can't find a suitable category?
+    af = Kan u nie 'n geskikte kategorie vind nie?
+    ar = لا يمكن العثور على فئة مناسبة؟
+    az = Uyğun kateqoriya tapa bilmirsiniz?
+    be = Не можаце знайсці прыдатную катэгорыю?
+    bg = Не можете да намерите подходяща категория?
+    ca = No trobeu una categoria adequada?
+    cs = Nemůžete najít vhodnou kategorii?
+    da = Kan du ikke finde en passende kategori?
+    de = Keine passende Kategorie gefunden?
+    el = Δεν μπορείτε να βρείτε την κατάλληλη κατηγορία;
+    es = ¿No encuentras una categoría adecuada?
+    et = Ei leia sobivat kategooriat?
+    eu = Ezin duzu kategoria egokirik aurkitu?
+    fa = آیا نمی توانید یک دسته بندی مناسب پیدا کنید؟
+    fi = Etkö löydä sopivaa luokkaa?
+    fr = Tu ne trouves pas de catégorie appropriée ?
+    he = לא מוצאים קטגוריה מתאימה?
+    hi = कोई उपयुक्त श्रेणी नहीं मिल रही?
+    hu = Nem talál megfelelő kategóriát?
+    id = Tidak dapat menemukan kategori yang sesuai?
+    it = Non riesci a trovare una categoria adatta?
+    ja = 適切なカテゴリーが見つからない？
+    ko = 적합한 카테고리를 찾을 수 없나요?
+    lt = Nerandate tinkamos kategorijos?
+    mr = योग्य श्रेणी शोधू शकत नाही?
+    nb = Finner du ikke en passende kategori?
+    nl = Kun je geen geschikte categorie vinden?
+    pl = Nie możesz znaleźć odpowiedniej kategorii?
+    pt = Não consegues encontrar uma categoria adequada?
+    pt-BR = Você não consegue encontrar uma categoria adequada?
+    ro = Nu găsiți o categorie potrivită?
+    ru = Не можешь найти подходящую категорию?
+    sk = Nemôžete nájsť vhodnú kategóriu?
+    sv = Kan du inte hitta en lämplig kategori?
+    sw = Je, huwezi kupata aina inayofaa?
+    th = ไม่พบหมวดหมู่ที่เหมาะสมใช่ไหม?
+    tr = Uygun bir kategori bulamıyor musunuz?
+    uk = Не можете знайти відповідну категорію?
+    vi = Không thể tìm thấy một danh mục phù hợp?
+    zh-Hans = 找不到合适的类别？
+    zh-Hant = 找不到合適的類別？
+
+  [editor_category_unsuitable_text1]
+    tags = android
+    en = Organic Maps does not support all categories. E.g. cities, railway stations, roads, lakes, or buildings can not be added. Please add such features directly via osm.org (use the Edit button in the top bar) or use a specialized OSM editing app.
```

Comment:
Maybe first sentence like
`Organic Maps allows to add simple point features only, that means no towns, roads, lakes, building outlines, etc.`
?

> Please add such features directly via osm.org (use the Edit button in the top bar) or use 

Here I read it like there is an Edit button in OM's top bar that'll get me to the right place in osm.org... Maybe drop the `(use the Edit button in the top bar)` completely? or rephrase

Reply:
"objects" then is clearer for newbies than a highly specific term "features".

Reply:
"objects", "map objects", "object types", etc. should be good indeed

Reply:
I second the suggestion for the first sentence, clarifying that only points are supported. Here's my slight edit to the suggestion:

```suggestion
    en = Organic Maps can only add point features, not lines or shapes. This means you cannot add roads, buildings, cities, railway stations, or lakes, among other things. Please add such features directly via osm.org (use the Edit button in the top bar of the website) or use a specialized OSM editing app.
```

I've also made roads and buildings first in the list of examples (they get the point across better than cities IMO) and clarified the top bar as noted.